dpnp/dpnp_iface_mathematical.py

```diff
@@ -704,7 +1054,27 @@ def trunc(x1, **kwargs):
     """
     Compute the truncated value of the input, element-wise.
 
-    .. seealso:: :obj:`numpy.trunc`
+    For full documentation refer to :obj:`numpy.trunc`.
+
+    Limitations
+    -----------
+        Parameter ``x1`` is supported as :obj:`dpnp.ndarray`.
+        Keyword arguments ``kwargs`` are currently unsupported.
+        Otherwise the functions will be executed sequentially on CPU.
+        Input array data types are limited by supported DPNP :ref:`Data types`.
+
+    See Also
+    --------
+        :obj:`dpnp.floor` : Round a number to the nearest integer toward minus infinity.
+        :obj:`dpnp.ceil` : Round a number to the nearest integer toward infinity.
+
+    Examples
+    --------
+    >>> import dpnp as np
+    >>> a = np.array([-1.7, -1.5, -0.2, 0.2, 1.5, 1.7, 2.0])
+    >>> result = np.trunc(a)
+    >>> [x for x in result]
+    [-1.0, -1.0, -0.0, 0.0, 1.0, 1.0, 2.0]
```

Comment:
We are creating another object for printing of `result`'s items. Isn't it better to just call print? I mean `print(results)`.

Reply:
`print` provides extra unnecessary information something like this `1 2 3 <__str__ TODO>`. So we selected this approach `[i for i in res]`.

Reply:
This is the known behavior for `dpnp.dparray`. It will be fixed after update of `__repr__` and `__str__` for `dpnp.dparray`.
I think, in any case, we can include here print result without `<__str__ TODO>` for one dim array. Otherwise the given approach may be confused.

Reply:
I vote for updating all the docstrings (if it will be required) after updating of `__repr__` and `__str__` for `dpnp.dparray` because already all the docstrings (except docstrings of the module `random`) use this approach.